sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/java/com/azure/ai/inkrecognizer/InkStrokeKind.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.azure.ai.inkrecognizer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The StrokeKind enum represents the class a stroke belongs to.
+ * The user of the Ink recognizer service is expected to set this value
+ * when it is known with absolute certainty. The default value is "UNKNOWN"
```

Comment:
The default is either UNKNOWN or "unknown", not "UNKNOWN". See why I think the converter needs to lowercase any string when doing a lookup? ;-)

Reply:
Trying to think what would happen if I apply this for Shapes enum too which has mixed case names.

Reply:
I think you can just put the fully qualified enum name here.

Reply:
The simple pattern is to lowercase all entries when the dictionary is created, and lowercase everything when looking things up

Reply:
Thanks Fred, modified as suggested.